src/debug_wait.h

```diff
@@ -0,0 +1,54 @@
+/*
+ * This file and its contents are licensed under the Apache License 2.0.
+ * Please see the included NOTICE for copyright information and
+ * LICENSE-APACHE for a copy of the license.
+ */
+
+#ifndef TIMESCALEDB_DEBUG_WAIT_H_
+#define TIMESCALEDB_DEBUG_WAIT_H_
+
+#include <postgres.h>
+
+#include <storage/lock.h>
+
+/* Debug waitpoint tag for debug waitpoints.
```

Comment:
Does it makes sense to call it breakpoint?

Reply:
Well... it does not really "break" execution, it just causes it to "wait", which is why I called it "waitpoint". I'm fine with changing the name though, if there are some good suggestions.